### PR TITLE
Create initial component structure

### PR DESCRIPTION
**Pull request by Thenlie**
- Create basic search form using react router Form
- Create movie card component. 
- Refine code.

### CODE REVIEW

**Comment by Thenlie** [comment]
📄 Draft PR created so other's can leave feedback on the component structure. Let me know what else should be added or changed. 

❌ #46 Will also kind of block this as a few components will be made in that ticket. Because of this I might wait until that has been completed to ensure we do not miss anything.

**Review by maxh1231 — DISMISSED** [review]

File: src/main.tsx [hunk 1]
```diff
@@ -34,7 +31,12 @@ const router = createBrowserRouter([
 			{
 				path: 'dashboard',
 				element: <DashboardScreen />
-			}
+			},
+			{
+				path: 'search',
+				element: <SearchResultsScreen />,
+				loader: searchLoader
+			}
```
Comment (maxh1231) [inline src/main.tsx:39]: `loader` function will return the query as a string, then attach it to the URL `/search/loader-string`?

**Review by maxh1231 — APPROVED** [review]
Components on components!

File: src/main.tsx (hunk 1 above)
Comment (Thenlie) [reply src/main.tsx:39]: > `loader` function will return the query as a string, then attach it to the URL `/search/loader-string`?

No, loader will return the string to the loaderData in the search route. Then it is accessed inside the component with `useLoaderData`. The loader is actually pulling the query from the URL.

**Pull request merged**